.github/ISSUE_TEMPLATE/defect.yml

```diff
@@ -0,0 +1,41 @@
+---
+name: Defect
+description: Report a defect, such as a bug or regression.
+labels:
+  - defect
+body:
+  - type: textarea
+    id: versions
+    attributes:
+      label: What version were you using?
+      description: Include the server version (`nats-server --version`) and any client versions when observing the issue.
```

Comment:
`...and the CLI version (nats --version)....` not sure new people would really grok client==cli in this case, this would be fine in client repos though

Reply:
Updated.